### PR TITLE
MINOR: Fix doc for producer throttle time metrics

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@junrao The doc doesn't match the implementation. I think the doc was always wrong since throttle time was never a topic metric. Is that correct? Thanks.

File: docs/ops.html [hunk 1]
```diff
@@ -1272,7 +1272,7 @@
 All the following metrics have a recording level of ``debug``:
  <table class="data-table">
       <tbody>
-      <tr>
+      tr>
```
Comment (ijuma) [inline docs/ops.html]: This doesn't look intentional

Comment (rajinisivaram) [reply docs/ops.html]: @ijuma Oops, sorry, fixed. Thank you!

**Comment by junrao** [comment]
@rajinisivaram : Thanks for the patch. LGTM

**Pull request closed without merging**